servers/lib/DEVELOPER.md

```diff
@@ -140,7 +130,7 @@ docker compose -f compose.lib.dev.yml up -d
 
 This command brings up the lib docker container and makes
 the website available at <http://localhost:4001>.
-The `config/.env.default` file is used as the microservice configuration.
+The `config/libms.dev.yaml` file is used as the microservice configuration.
```

Comment:
The volume mapping of `servers/lib/compose.lib.dev.yml` need to be updated to reflect this use.

Reply:
@prasadtalasila  But I guess this depends on the ${LIB_CONFIG}?

Reply:
Please do